hack/list-env-vars.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Useful to list all the environment variables that are used in the code, in alphabetical order.